notifications_utils/clients/redis/bounce_rate.py

```diff
@@ -78,4 +83,26 @@ def get_bounce_rate(self, service_id: str, bounce_window=_twenty_four_hour_windo
             total_notifications_key(service_id), min_score=twenty_four_hours_ago, max_score=now
         )
 
-        return round(total_hard_bounces / (1.0 * total_notifications), 2) if (total_notifications > 0) else 0.0
+        if total_notifications < 1:
+            return 0.0
+
+        return round(total_hard_bounces / (1.0 * total_notifications), 2)
```

Comment:
Same here - we were getting a "divide by zero" error because something like `0.00000000001` was getting through the `total_notifications > 0` comparison